application/tests/api/ResetCest.php

```diff
@@ -137,6 +137,13 @@ public function test8(ApiTester $I)
         $I->seeResponseCodeIs(200);
     }
 
+    public function test81(ApiTester $I)
+    {
+        $I->wantTo('check response when making unauthenticated PUT request to validate an expired reset code');
```

Comment:
I'm not sure which this is doing, but it may be worth testing our handling of both an expired reset when given the correct code and an expired reset when given an incorrect code.

Reply:
I added another test for this.